index.html

```diff
@@ -185,42 +184,50 @@ <h3>Skills</h3>
                     </ul>
                 </div>
             </section>
-        </div>
 
-        <div class="wrapper-menu" id="wrapper-menu">
-            <header class="header">
-                <nav class="nav">
-                    <img src="img/image_geometry_menu_1.svg" alt="">
-                    <a href="#" onclick="closeMenu()"><img class="menu-button" src="img/Union.svg" alt="Menu Buttom"></a>
-                </nav>
-            </header>
+            <section class="contact">
+                <div class="contact-form">
+                    <div class="title set">
+                        <h2>Get started</h2>
+                    </div>
+                    <div class="content set">
+                        <p class="description">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.
+                        </p>
+                        <form class="user-form" action="https://formspree.io/f/xoqrloon" method="post">
+                            <label for="name">
+                                Your name
+                                <input class="name-font" type="text" name="name" id="name" maxlength="30" value="Anastasiia">
+                            </label>
+                            <label for="email" id="email_label">
+                                Your email
+                                <input type="email" name="email" id="email" placeholder="Enter your email address" required>
+                            </label>
+                            <textarea id="message" name="message" placeholder="Enter your message here..." required></textarea>
```

Comment:
- [ ] Please ensure that the `textarea` input has a `maxlength` property of value `500` as required.